pkg/ccl/importccl/import_stmt.go

```diff
@@ -953,6 +953,18 @@ func importPlanHook(
 			return nil
 		}
 
+		// Double check that this transaction is implicit, so that
+		// we're allowed to commit.
+		if !p.ExtendedEvalContext().TxnImplicit {
+			return errors.AssertionFailedf("expected import planning transaction to be implicit")
+		}
+		// We're about to create the job record. We don't want this transaction
```

Comment:
More generally: we're about to have some side-effects that are not tied to this txn, which is okay OK if this txn _will_ commit and the only way to be sure of that is to commit it, which we're allowed to do since it is implicit.

Reply:
Updated the comment.